pkg/authorization/client/clientset_generated/internalclientset/clientset.go

```diff
@@ -0,0 +1,73 @@
+package internalclientset
+
+import (
+	"github.com/golang/glog"
+	v1core "github.com/openshift/origin/pkg/authorization/client/clientset_generated/internalclientset/typed/core/v1"
+	restclient "k8s.io/kubernetes/pkg/client/restclient"
+	discovery "k8s.io/kubernetes/pkg/client/typed/discovery"
+	"k8s.io/kubernetes/pkg/util/flowcontrol"
+)
+
+type Interface interface {
+	Discovery() discovery.DiscoveryInterface
```

Comment:
@smarterclayton will public `var LegacyPrefix` do or we can do smarter plumbing?